utils/src/main/java/ai/elimu/common/utils/ContextExt.kt

```diff
@@ -24,14 +24,24 @@ fun Context.isPackageInstalled(packageName: String,
             .setMessage(dialogMessage)
             .setPositiveButton(buttonText
             ) { _, _ ->
-                val openProviderIntent = Intent().apply {
+                val openAppStoreIntent = Intent().apply {
```

Comment:
@tuancoltech Is this utility code meant to work with _any_ application package name? If so, we should probably rename this variable from "openAppStore" to something else?

Reply:
> Is this utility code meant to work with any application package name
@jo-elimu Yes. I've added documentation for this function to clarify its usage, as well as renaming the intent variable name.